test/metabase/models/setting_test.clj

```diff
@@ -97,6 +98,13 @@
   :default    "setting-default"
   :feature    :test-feature)
 
+(defsetting test-never-encrypted-setting
```

Comment:
Could maybe also include a basic test that a `:boolean` setting defaults to `:never`?